src/OrchardCore.Modules/OrchardCore.Lists/Drivers/ListPartDisplayDriver.cs

```diff
@@ -35,9 +43,9 @@ public override IDisplayResult Display(ListPart listPart, BuildPartDisplayContex
                     {
                         var pager = await GetPagerSlimAsync(context);
                         var settings = context.TypePartDefinition.GetSettings<ListPartSettings>();
-
+                        var listpartFilter = new ListPartFilter();
```

Comment:
A controllerhez hasonlóan itt is csak simán `null`-t adhatnál át egy üres object helyett (vagy ha összevonod a `publishedOnly`-val, ahogy írtam, akkor kelleni fog).

Reply:
Továbbra is.

Reply:
Továbbra is. Kérlek mondd el, miért resolve-olod, ha nem oldottad meg.

Reply:
Félreértettem, mivel felül azt írtad "vagy ha összevonod a publishedOnly-val, ahogy írtam, akkor kelleni fog" 
úgy értelmeztem szükség lesz a kijelölt listpartFilter, mostani nevén, containedItemOptions-ra. 
De most már töröltem és null-t adtam át utolsó paraméternek.

Reply:
Jelenleg nincs használva semmire sem, szóval a mostani állapot nem jó sehogy se. Úgyhogy vagy nem kell a példány, vagy kell, de akkor fel kell benne tölteni a statust is.

Reply:
Rendben, szerintem fenti metódusban nem lesz rá szükség.